rocAL/rocAL/CMakeLists.txt

```diff
@@ -120,6 +124,11 @@ if(NOT Threads_FOUND)
     set(BUILD_ROCAL false)
     message("-- ${Yellow}NOTE: rocAL library requires Threads, Not Found ${ColourReset}")
 endif()
+if (${libsnd_LIBS} STREQUAL "libsnd_LIBS-NOTFOUND")
```

Comment:
Check if rocal build should be set to false if sndlib is not found

Reply:
Set it to false when libsnd is not found